tests/mock-data.js

```diff
@@ -25,10 +26,13 @@ mock.exampleDocWithInvocation.beta =
   require('./mock-documents/example-doc-with-beta-invocation');
 
 const didContext = require('./mock-documents/did-context');
-_loaderData['https://w3id.org/did/v0.11'] = didContext;
+_loaderData.set('https://w3id.org/did/v0.11', didContext);
 
 const v1Context = require('./mock-documents/veres-one-context');
-_loaderData['https://w3id.org/veres-one/v1'] = v1Context;
+_loaderData.set('https://w3id.org/veres-one/v1', v1Context);
+
+const {suiteContext} = require('@digitalbazaar/ed25519-signature-2018');
```

Comment:
Are we continuing to support 2018 here?

Reply:
I don't know, I was just getting this working, it didn't actually work before with the new zcap context.